brave/brave-core
# Added VPN usage P3A questions

Resolves brave/brave-browser#23892

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

On Windows:

0. Start with fresh profile with VPN feature enabled.
1. Ensure `Brave.VPN.NewUserReturning` has value 0 in local state page.
2. Turn on VPN
3. Ensure "new user returning" metric has a value of 2, "days in month" metric has a value of 1, and "last usage time" has a value of 1
4. Close browser, advance system date by one day, open browser.
5. Ensure all values in step 3 are current.
6. Turn on VPN
7. Ensure "new user returning" metric has a value of 3, "days in month" metric has a value of 2, and "last usage time" has a value of 1
8. Close browser advance system date by one week, open browser.
9. Ensure "new user returning" metric has a value of 1, "days in month" metric has a value of 2, and "last usage time" has a value of 2
10. Turn on VPN
11. Ensure "new user returning" metric has a value of 1, "days in month" metric has a value of 3, and "last usage time" has a value of 1

On Android:

0. Start with fresh profile with VPN feature enabled.
1. Ensure `Brave.VPN.NewUserReturning` has value 0 in local state page.
2. Turn on VPN
3. Ensure "new user returning" metric has a value of 2, "days in month" metric has a value of 1, and "last usage time" has a value of 1
4. Close browser, ensure VPN is still enabled in the background, advance system date by two days, open browser.
5. Ensure "new user returning" metric has a value of 3, "days in month" metric has a value of 3, and "last usage time" has a value of 1
6. Turn off VPN, close browser, advance system date by one week, open browser.
7. Ensure "new user returning" metric has a value of 1, "days in month" metric has a value of 3, and "last usage time" has a value of 2
